src/components/VmDetails/cards/SnapshotsCard/style.css

```diff
@@ -76,7 +76,8 @@ div.popover {
 
 .snapshot-item-time {
   color: #808080;
-  padding-left: 5px;
+  display: block;
```

Comment:
This will make ALL snapshot times appear below the description, even when there is plenty of space for the time.
![screenshot-chopper local dickerson-web com-2019 06 13-17-07-15](https://user-images.githubusercontent.com/3985964/59467468-c140c300-8dfd-11e9-9b6b-2c20cb67b1ba.png)

Perhaps you should use `inline-block`?

And the 5px padding between the name and the time should still be there...maybe a `padding-right: 5px` on the name, or just leave it in there and let the time push over to the right if the block has to flow to the next line...